.travis.yml

```diff
@@ -1,23 +1,26 @@
 dist: trusty
 sudo: required
-language: node_js
-node_js:
-- "6"
+language: python
+python:
+- "3.6"
```

Comment:
I think we've standardized on python 3.5, at least in terms of what we're deploying to the BBBs. So I don't think it makes sense to use a different version for travis.